src/test/resources/integration/listUsers-request.txt

```diff
@@ -0,0 +1,3 @@
+curl -X "GET" "https://api.sandbox.hyperwallet.com/rest/v3/users" \
```

Comment:
Pls add some query parameters for filtering options here.

Reply:
As discussed before this couldnt be done as testing dynamically with query parameters is not feasible in the current mock test setup.